app/controllers/planning_applications_controller.rb

```diff
@@ -70,12 +68,9 @@ def update
 
   def assign
     if request.patch?
-      @planning_application.user = if params[:planning_application][:user_id] == "0"
-                                     nil
-                                   else
-                                     current_local_authority.users.find(params[:planning_application][:user_id])
-                                   end
-      audit("assigned", nil, @planning_application.user&.name)
+      user = params[:planning_application][:user_id].eql?("0") ? nil : current_local_authority.users.find(params[:planning_application][:user_id])
```

Comment:
I know this was from before but this just looks a bit wrong getting the user like this haha. I think this whole controller action needs a rethink

Reply:
Agreed can be on a later PR